ios/FluentUI/BarButtonItems/BarButtonItems.swift

```diff
@@ -10,7 +10,7 @@ public typealias MSBarButtonItems = BarButtonItems
 
 public final class BarButtonItems: NSObject {
     /// When adding this barButtonItem to the view, tint it with appropriate app color UIColor(light: Colors.primary(for: window), dark: Colors.textDominant)
-    @objc static func confirm(target: Any?, action: Selector?) -> UIBarButtonItem {
+    @objc public static func confirm(target: Any?, action: Selector?) -> UIBarButtonItem {
```

Comment:
same here.  it was purposeful to reduce teh number of public extension in the library. the demo app should have its own custom barbutton item to illustrate the usage

Reply:
Restored original BatButtonItems class and moved this code in Demo project

Reply:
Moved it to Demo project